corehq/apps/app_manager/tests/test_tasks.py

```diff
@@ -14,6 +16,11 @@
 
 @patch_validate_xform()
 class AppManagerTasksTest(AppManagerTest):
+
+    def tearDown(self):
+        CaseType.objects.filter(domain=self.domain).delete()
```

Comment:
This is unnecessary since `django.test.TestCase` will roll back the SQL transaction that is started for each test.

Reply:
Thank you, I always forget that. Updated in c2613c8f6ad